app/battle/knight_creator.py

```diff
@@ -0,0 +1,85 @@
+class Knight:
+
+    def __init__(
+            self,
+            knight_nik: str,
+            knight_params: dict,
+            list_order: int
+    ) -> object:
+
+        """
+        Args:
+            knight_nik (str):
+            The nickname of the Knight.
+            knight_params (dict):
+            The dictionary containing the parameters of the Knight.
+            list_order (int):
+            The position of the Knight in the list.
+        Returns:
+            object: The Knight object.
```

Comment:
You don't need to add explanation when login is understandable just by reading the code. Remove it

Reply:
Fix everywhere